### ISSUE TITLE
Cant update delay or transitions

### ISSUE THREAD

**Issue by NMonst4** [deulos/vue-flux#5]
Hi,

very good vue component that saved me a lot of time but now i have some little problems.

I update the delay or transition of the fluxOptions Object or the fluxTransitions Object but the changes would not be recognized by the vue-flux component.

For example i add a button that changes the delay from 5000 to 10000 but the delay didnt changed.

Regards, NMonst4

**Comment by NMonst4** [comment]
close :D my last try was

`this.$refs.slider.options.delay` 

But YES this works well. Ok now only the last thing i need is the change of transition by a button i think you have done that in your example right?

**Comment by deulos** [comment]
About transitions you will not be able to do the same, because it reads the names of transitions when the slider is created.

In case you just want to trigger a transition you can do it like in the example with:
``` javascript
$refs.slider.showImage('next', 'transitionBars3d')
```

I'll add the method to update transitions 👍

**Comment by NMonst4** [comment]
Sounds great . Thank you

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by deulos** [comment]
If I don't do any more updates this week I will release the version 3.0.0 with the new methods at the end of the week :)

**Comment by NMathar** [comment]
Hi Deulos,

i think you added the methods setOptions and setTransitions for this purpose right? 

If i set the new delay options like this

````javascript
                this.$refs.slider.setOptions({
                    delay: 100
                });
````
i get massive errors from your component. 

Is this the right way to update the options now or i am doing it false?

Error
`
[Vue warn]: Error in render: "RangeError: Invalid array length"

found in

---> <FluxGrid> at FluxGrid.vue
       <TransitionTurn3d> at blinds2d.vue
         <VueFlux> at VueFlux.vue
`

Regards Nico

**Comment by deulos** [comment]
Yes, shouldnt give you an error, I will check it and fix it today :).

Sorry for the inconvenience.

**Comment by deulos** [comment]
I found the problem, 2 lines were out of place... I will release the npm package this afternoon

**Comment by deulos** [comment]
It is released.

Regards!

**Comment by NMathar** [comment]
Thank you <3 it works like a charm. :)
